sample-extension-plugin/build.gradle

```diff
@@ -44,3 +44,17 @@ testClusters.integTest {
     // need to install job-scheduler first, need to assemble job-scheduler first
     plugin file("${rootProject.getBuildDir()}/distributions/${rootProject.getName()}-${project.getVersion()}.zip")
 }
+
+// As of ES 7.7 the sample-extension-plugin is being added to the list of plugins for the testCluster during build before
+// the job-scheduler plugin is causing build failures.
+// The job-scheduler zip is added explicitly above but the sample-extension-plugin is added implicitly at some time during evaluation.
+// Will need to do a deep dive to find out exactly what task adds the sample-extension-plugin and add job-scheduler there but a temporary hack is to
+// reorder the plugins list after evaluation but prior to task execution when the plugins are installed.
+afterEvaluate {
```

Comment:
Can we make this a bit more reliable by checking if the sample plugin comes before job scheduler plugin and if so then removing/adding? Otherwise if it changes again next version this will break again.

Reply:
I think I can make this change in this version. There is no doubt it will make the plugin installation order much more reliable, but is not that important. The whole `afterEvaluate` block is a temporary workaround, and I believe the plugins should not be installed in this way forever, so the `afterEvaluate` block won't last long. An ideal case is in next version, the `testClusters` Gradle plugin get changed and this problem get fixed, and this gradle file get restored.

Reply:
Looking forward to hear opinions from other reviewers as well.